nuxeo-core/nuxeo-core-test/src/test/java/org/nuxeo/ecm/core/bulk/DocumentSetRepositoryInit.java

```diff
@@ -28,15 +28,47 @@
  */
 public class DocumentSetRepositoryInit extends DefaultRepositoryInit {
 
-    private static final int SIZE = 10;
+    public static final String ROOT = "/default-domain/workspaces/test";
+
+    public static final int SIZE = 3;
```

Comment:
can't you state how much document or proxy you are going to create instead of size and level?